src/components/Formulario.js

```diff
@@ -0,0 +1,57 @@
+import React, {Fragment, useState} from 'react';
+
+
+
+const Formulario = () => {
+
+
+    const [datos, setDatos] = useState({
+        usuario: '',
+        contraseña: ''
+        
+    })
+
+    const SetData = (event) => {
+
+         // console.log(event.target.name)
+         // console.log(event.target.value)
+         // console.log(event.target.placeholder)
+       setDatos({
+           ...datos,
+           [event.target.name]: event.target.value
+       })
+
+    }
+
+    const enviarDatos = (event) => {
+        event.preventDefault()
+
+        if (datos.usuario === 'teamhorda' && datos.contraseña === "123" ) {
+           alert('enviando datos...' + datos.usuario + ' ' + datos.contraseña)
+           alert("se valido su usuario")
+        }else{
+             alert("no valido su usuario")
+
+        }
+        
+    }
+
+
+    return(
+        <Fragment>
```

Comment:
Do you understand the Fragment directive? please remove it and add a div instead